projects/laji/src/app/shared/service/confirm-modal.component.ts

```diff
@@ -27,16 +27,16 @@ import { BsModalRef } from 'ngx-bootstrap/modal';
 })
 export class ConfirmModalComponent implements OnInit, AfterViewInit {
 
-  message: string;
+  message!: string;
   confirmLabel = 'OK';
   cancelLabel = 'cancel';
   prompt = false;
   promptValue = '';
   showCancel = true;
 
-  value: boolean | string | null;
+  value!: boolean | string | null;
```

Comment:
I'm wondering what does the exclamation mark means here? I thought that it means that we tell the compiler that it can't be null but then its type has null?